addons/l10n_es_edi_tbai/models/l10n_es_edi_tbai_certificate.py

```diff
@@ -2,18 +2,60 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from base64 import b64decode
+from pytz import timezone
+from datetime import datetime
+from cryptography.hazmat.primitives.serialization import Encoding, NoEncryption, PrivateFormat
 
-from odoo import models
+
+from odoo import _, api, fields, models, tools
+from odoo.exceptions import ValidationError
 from odoo.addons.account.tools.certificate import load_key_and_certificates
 
 
 class Certificate(models.Model):
-    _inherit = 'l10n_es_edi.certificate'
+    _name = 'l10n_es_edi_tbai.certificate'
+    _description = 'Personal Digital Certificate'
+    _order = 'date_start desc, id desc'
+    _rec_name = 'date_start'
+
+    content = fields.Binary(string="File", required=True, help="PFX Certificate")
+    password = fields.Char(help="Passphrase for the PFX certificate", groups="base.group_system")
```

Comment:
note that the model is already restricted to the group_system so the groups doesn’t change anything. If you want that no-one (except sudo) access this field, there is the `fields.NO_ACCESS`

Reply:
Thx a lot for the review :)

I've removed the groups attribute.
The code is copy-pasted from the sii certificate, so i don't know why it was written that way (I also changed for that certificate as it has the same problem).
https://github.com/odoo/odoo/blob/92db8ec761e5b36c08cd59d46d3f907feef58466/addons/l10n_es_edi_sii/models/l10n_es_edi_certificate.py#L22